src/test/java/com/squareup/javapoet/ClassNameTest.java

```diff
@@ -193,4 +193,14 @@ public void reflectionName() {
     assertEquals("Foo$Bar$Baz", ClassName.get("", "Foo", "Bar", "Baz").reflectionName());
     assertEquals("a.b.c.Foo$Bar$Baz", ClassName.get("a.b.c", "Foo", "Bar", "Baz").reflectionName());
   }
+
+  @Test
+  public void canonicalName() {
+    assertEquals("java.lang.Object", TypeName.OBJECT.canonicalName());
```

Comment:
Nice tests!

Reply:
Is it some kind of approve or necessary to do something?